clean_md_commodities.py

```diff
@@ -22,287 +22,298 @@
+df3=df3.replace(to_replace='sercite', value='sericite', regex= True)
+df3=df3.replace(to_replace='molyhdenum',value='molybdenum', regex= True)
+df3=df3.replace(to_replace='golq', value='gold', regex= True)
+df3=df3.replace(to_replace='coopper', value='copper', regex= True)
+df3=df3.replace(to_replace='copper;bairite', value='copper; barite', regex= True)
+df3=df3.replace(to_replace='posellite', value='powellite', regex= True)
+df3=df3.replace(to_replace='tellinium', value='tellurium', regex= True)
+df3=df3.replace(to_replace='silver-sinc-lead', value='silver-zinc-lead', regex= True)
+df3=df3.replace(to_replace='tungiten', value='tungsten', regex= True)
+df3=df3.replace(to_replace='siilver', value='silver', regex= True)
+df3=df3.replace(to_replace='laead', value='lead', regex= True)
+df3=df3.replace(to_replace='montmorillinitic clay', value='montmorillonite clay', regex= True)
+df3=df3.replace(to_replace='wallastonite', value='wollastonite', regex= True)
+df3=df3.replace(to_replace='modybdenum', value='molybdenum', regex= True)
+df3=df3.replace(to_replace='silve', value='silver', regex= True)
+df3=df3.replace(to_replace='sulphur deposites', value='sulphur', regex= True)
+df3=df3.replace(to_replace='tunsgten', value='tungsten', regex= True)
+df3=df3.replace(to_replace='talc;chlorite', value='talc; chlorite', regex= True)
+df3=df3.replace(to_replace='mersury', value='mercury', regex= True)
+df3=df3.replace(to_replace='sodium sufate', value='sodium sulfate', regex= True)
+df3=df3.replace(to_replace='sulphides', value='sulfides', regex= True)
+df3=df3.replace(to_replace='sulvides and precious metals', value='sulfides and precious metals', regex= True)
+df3=df3.replace(to_replace='uranium, copper, nickle, cobalt', value='uranium, copper, nickel, cobalt', regex= True)
+df3=df3.replace(to_replace='vanadiuml zinc', value='vanadium zinc', regex= True)
+df3=df3.replace(to_replace='diamtomite', value='diatomite', regex= True)
+df3=df3.replace(to_replace='fluroite;gold', value='fluroite; gold', regex= True)
+df3=df3.replace(to_replace='vanudium', value='vanadium', regex= True)
+df3=df3.replace(to_replace='siver', value='silver', regex= True)
+df3=df3.replace(to_replace='betonite', value='bentonite', regex= True)
+df3=df3.replace(to_replace='arsencic', value='arsenic', regex= True)
+df3=df3.replace(to_replace='chrysacolla', value='chrysocolla', regex= True)
+df3=df3.replace(to_replace='colbalt', value='cobalt', regex= True)
+df3=df3.replace(to_replace='volcanid ash', value='volcanic ash', regex= True)
+
+df4=pd.DataFrame(intersection)
+df4=df4.replace(to_replace='soduim nitrate', value='sodium nitrate', regex= True)
+df4=df4.replace(to_replace='sodium sufate', value='sodium sulfate', regex= True)
+df4=df4.replace(to_replace='leaseable minerals', value='leasable minerals', regex= True)
+
+df5=pd.DataFrame(comm_list_arr)

```

Comment:
I don't understand why you're replacing values in this array as well as the `lower_gen_comm` list. How are you going to reconcile them to make one master commodities list again?